pywbemtools/pywbemcli/_cmd_server.py

```diff
@@ -412,3 +437,138 @@ def cmd_server_remove_mof(context, options):
+
+    rows = []
+    detail = options['detail']
+
+    for ns in sorted(namespaces):
+        klasses = conn.EnumerateClasses(namespace=ns, DeepInheritance=True,
+                                        LocalOnly=True)
+        classes_count = len(klasses)
+        ns_experimental = False
+        ns_max_version = [0, 0, 0]
+        # Schema names should be case independent
+        schemas = NocaseDict()
+        schema_max_ver = NocaseDict()
+        schema_experimental = NocaseDict()
+        no_schema = []
+
+        for klass in klasses:
+            this_class_experimental = False
+            # Determine if experimental qualifier exists
+            if ns_experimental is False:
+                if is_experimental_class(klass):
+                    ns_experimental = True
+                    this_class_experimental = True
+
+            schema_elements = klass.classname.split('_', 1)
+            schema = schema_elements[0] if len(schema_elements) > 1 \
+                else "{no schema}"
+
+            schemas[schema] = schemas.get(schema, 0) + 1
+            if len(schema_elements) < 2:
+                no_schema.append(klass.classname)
+            if schema not in schema_max_ver:
+                schema_max_ver[schema] = [0, 0, 0]
+
+            # If detail, get the experimental qualifier for this schema
+            if detail:
+                if schema not in schema_experimental:
+                    schema_experimental[schema] = False
+                if this_class_experimental:
+                    schema_experimental[schema] = True
+                elif ns_experimental:
+                    if schema_experimental[schema] is False:
+                        if is_experimental_class(klass):
+                            schema_experimental[schema] = True
+
+            # Get the version qualifier for this class
+            if 'Version' in klass.qualifiers:
+                version = klass.qualifiers['Version'].value
+                version = parse_version_value(version, klass.classname)
+
+                if version > ns_max_version:
+                    ns_max_version = version
+
+                if detail:
+                    if schema not in schema_max_ver or \
+                            version > schema_max_ver[schema]:
+                        schema_max_ver[schema] = version
+
+        # Build the table formatted output
+        prev_namespace = None
+        ns_version_str = version_str(ns_max_version) if classes_count else ""
+
+        if detail:
+            # Display with a line for each namespace and one for each
+            # schema in the namespace
+            for schema in sorted(schemas.keys()):
+                schema_max_ver_str = version_str(schema_max_ver[schema])
+                # Append a row for each new namespace
+                if ns != prev_namespace:
+                    prev_namespace = ns
+                    rows.append([ns,
+                                 "(namespace-info)",
```

Comment:
Since the other rows show the schema name, it is a bit confusing to now talk about "namespace-info". I suggest to show "(all schemas)" or something along these lines

Reply:
DONE.  I could not think of anything better and a blank is confusing also

Reply:
In the current code as of commit 6702249, it still says "namespace-info". The suggested "--------" is similarly vague as showing blanks. It is a line with the totals for all schemas in this one namespace, so I could go with any of these (by decreasing priority):
* "(totals)"
* "(all)"
* "(all schemas)"
* "(namespace)"

Reply:
Slelected (totals) DONE